packages/wrangler/src/versions/upload.ts

```diff
@@ -112,6 +125,282 @@ function errIsStartupErr(err: unknown): err is ParseError & { code: 10021 } {
+			alias: "compatibility-flag",
+			type: "string",
+			requiresArg: true,
+			array: true,
+		},
+		latest: {
+			describe: "Use the latest version of the Worker runtime",
+			type: "boolean",
+			default: false,
+		},
+		assets: {
+			describe: "Static assets to be served. Replaces Workers Sites.",
+			type: "string",
+			requiresArg: true,
+		},
+		format: {
+			choices: ["modules", "service-worker"] as const,
+			describe: "Choose an entry type",
+			deprecated: true,
+			hidden: true,
+		},
+		"legacy-assets": {
+			describe: "Static assets to be served",
+			type: "string",
+			requiresArg: true,
+			deprecated: true,
+			hidden: true,
+		},
+		site: {
+			describe: "Root folder of static assets for Workers Sites",
+			type: "string",
+			requiresArg: true,
+			hidden: true,
+			deprecated: true,
+		},
+		"site-include": {
+			describe:
+				"Array of .gitignore-style patterns that match file or directory names from the sites directory. Only matched items will be uploaded.",
+			type: "string",
+			requiresArg: true,
+			array: true,
+			hidden: true,
+			deprecated: true,
+		},
+		"site-exclude": {
+			describe:
+				"Array of .gitignore-style patterns that match file or directory names from the sites directory. Matched items will not be uploaded.",
+			type: "string",
+			requiresArg: true,
+			array: true,
+			hidden: true,
+			deprecated: true,
+		},
+		var: {
+			describe: "A key-value pair to be injected into the script as a variable",
+			type: "string",
+			requiresArg: true,
+			array: true,
+		},
+		define: {
+			describe: "A key-value pair to be substituted in the script",
+			type: "string",
+			requiresArg: true,
+			array: true,
+		},
+		alias: {
+			describe: "A module pair to be substituted in the script",
+			type: "string",
+			requiresArg: true,
+			array: true,
+		},
+		"jsx-factory": {
+			describe: "The function that is called for each JSX element",
+			type: "string",
+			requiresArg: true,
+		},
+		"jsx-fragment": {
+			describe: "The function that is called for each JSX fragment",
+			type: "string",
+			requiresArg: true,
+		},
+		tsconfig: {
+			describe: "Path to a custom tsconfig.json file",
+			type: "string",
+			requiresArg: true,
+		},
+		minify: {
+			describe: "Minify the Worker",
+			type: "boolean",
+		},
+		"upload-source-maps": {
+			describe:
+				"Include source maps when uploading this Worker Gradual Rollouts Version.",
+			type: "boolean",
+		},
+		"node-compat": {
+			describe: "Enable Node.js compatibility",
+			type: "boolean",
+		},
+		"dry-run": {
+			describe: "Don't actually deploy",
+			type: "boolean",
+		},
+		tag: {
+			describe: "A tag for this Worker Gradual Rollouts Version",
+			type: "string",
+			requiresArg: true,
+		},
+		message: {
+			describe:
+				"A descriptive message for this Worker Gradual Rollouts Version",
+			type: "string",
+			requiresArg: true,
+		},
+	},
+	handler: async function versionsUploadHandler(args, { config }) {
```

Comment:
I don't think we can use this config as-is—this command used `const configPath =
		args.config ||
		(args.script && findWranglerConfig(path.dirname(args.script)));` to locate the config, logic which isn't in defineCommand

Reply:
Updated here: https://github.com/cloudflare/workers-sdk/pull/7437/commits/c447e879c97b5a0b435e2926dbff2413e998833e